osu.Game/Online/API/Requests/Responses/Comment.cs

```diff
@@ -40,11 +38,16 @@ public class Comment
         [JsonProperty(@"votes_count")]
         public int VotesCount { get; set; }
 
-        [JsonProperty(@"commenatble_type")]
-        public string CommentableType { get; set; }
+        public CommentableType CommentableType;
+
+        [JsonProperty(@"commentable_type")]
+        private string commentableType
+        {
+            set => Enum.TryParse(value.Pascalize(), true, out CommentableType);
```

Comment:
Used this as a reference
https://github.com/ppy/osu/blob/db34ea88bc26626b8ffc0e0e64f00e1507f2c455/osu.Game/Online/API/Requests/Responses/APIKudosuHistory.cs#L56

Reply:
Using your suggestion `CommentableType` for news post is trying to be `news_post`, when it must be `NewsPost`

Reply:
I'd see a use for having a converter similar to `StringEnumConverter` that does this process for us, since this isn't going to be the only place where this is done.